docs/mirroring/README.md

```diff
@@ -109,7 +110,13 @@ Instead, link to existing official guides and merely provide a lightweight point
 See: https://kubernetes.io/docs/contribute/style/content-guide/#dual-sourced-content
 -->
 
-<!--TODO: cri-o, general manifests-->
+<!--TODO: cri-o-->
+
+A simple option for many cases is to update the `image` fields in your 
```

Comment:
self-nit post-merge: I think we stopped using "simple" in docs because it assumes level of expertise etc.

will clean that up in a follow-up

Reply:
Yep: https://kubernetes.io/docs/contribute/style/style-guide/#avoid-words-that-assume-a-specific-level-of-understanding

Reply:
https://github.com/kubernetes/registry.k8s.io/pull/202